client/src/components/ClientProfile/Information/Information.js

```diff
@@ -16,96 +13,190 @@ import VisibilityIcon from '@material-ui/icons/Visibility';
 import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
-// Local changes
+// Modal sweetalert2
+import Swal from 'sweetalert2';
```

Comment:
Why don't you use `components/Modal`? It is used by material and you can use it.

Reply:
It's optional like snackbar from Material UI, you know.